templates/Dockerfile-deb.tmpl

```diff
@@ -57,22 +57,25 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main' >
         libxslt-dev \
         lintian \
         postgresql-server-dev-all \
-        wget \
+        # Add 12 package explicitly as *-all does not yet depend on 12 yet
```

Comment:
This line is hopefully temporary. Maybe rewrite as _As of Oct. 28, we still must add server-dev-12 as server-dev-all does not yet depend on it_. Sure, it'll be two lines but it'll explain the timeline better.

Reply:
Tested that the issue was temporary. Removing the extra lines and commments, as `postgresql-server-dev-all` is sufficient now